x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -261,7 +269,17 @@ export class TaskManagerRunner implements TaskRunner {
 
     try {
       this.task = this.definition.createTaskRunner(modifiedContext);
-      const result = await withSpan({ name: 'run', type: 'task manager' }, () => this.task!.run());
+      const ctx = {
+        type: 'task manager',
+        name: `run ${this.instance.task.taskType}`,
+        id: this.instance.task.id,
+        description: 'run a task',
+        // is there a way to create a URL to navigate to a task
+        // url: '',
```

Comment:
Unlike alerting rules, there are no useful URLs for individual tasks.  Could be a future task manager extension - provide a URL to thread into this call.

We **DO** have a health API available via fixed URL, which we could provide.  I guess the question is whether this field is intended to be a navigation to UI, or to provide a URL that provided specific information about the object (task in this case), or it can be anything useful (like, perhaps, the task manager health API).

Reply:
> We DO have a health API available via fixed URL, which we could provide. I guess the question is whether this field is intended to be a navigation to UI, or to provide a URL that provided specific information about the object (task in this case),

It depends on the nature of a Kibana entity triggering a request. 
If it's something created by a user (a Dashboard or a Visualization), an URL should navigate to this entity and ideally allow the user to edit it to reduce the query performance overhead.
`task` is an internal concept of Kibana, so the Kibana user can adjust `task` parameters neither in UI nor via an HTTP endpoint. In this case, `url` might be omitted.